modules/4_detection/test/test_detector.cpp

```diff
@@ -58,11 +58,11 @@ TEST(detection, faces) {
         Rect(319, 210, 19, 27), Rect(31, 238, 23, 30)
     };
     std::vector<float> refProbs = {
-        0.988363, 0.987432, 0.983568, 0.981547, 0.980774, 0.980398,
-        0.97681,  0.972978, 0.964593, 0.960844, 0.958782, 0.942152,
-        0.913904, 0.903534, 0.888043, 0.879184, 0.875234, 0.862958,
-        0.815154, 0.810955, 0.797403, 0.792146, 0.771417, 0.710467,
-        0.696174, 0.619217, 0.577113, 0.418896, 0.306871
+        0.988363f, 0.987432f, 0.983568f, 0.981547f, 0.980774f, 0.980398f,
+        0.97681f,  0.972978f, 0.964593f, 0.960844f, 0.958782f, 0.942152f,
+        0.913904f, 0.903534f, 0.888043f, 0.879184f, 0.875234f, 0.862958f,
+        0.815154f, 0.810955f, 0.797403f, 0.792146f, 0.771417f, 0.710467f,
+        0.696174f, 0.619217f, 0.577113f, 0.418896f, 0.306871f
```

Comment:
Can you please open a separate PR with that fix? Might be useful

Reply:
You also need to keep this file unchanged otherwise the system will run Detection model tests as well